src/collections/blog/2022-01-14-open-source-journey-gaurav-chadha/index.mdx

```diff
@@ -19,7 +19,6 @@ import { BlogWrapper } from "../Blog.style.js";
 import Profile from "./Intern-profile.png";
 
 <BlogWrapper>
-<img src={Profile} align="right" alt="profile-img"  width="380" height="500" />
 <p>
   Hey👋, I am Gaurav Chadha, a student from New Delhi, India with lots of dreams, ambitions, and zeal to learn and grow more each and every day.
 </p>
```

Comment:
@Ardent10 remove this file changes as it belongs to some other issue and PR is already opened for this one.